input/fsh/valueSets.fsh/unordered.fsh

```diff
@@ -0,0 +1,227 @@
+* ^identifier.value = "/orgs/Kenya_SMART_Guidelines/ValueSet/HIV.D.DE7/"
+* ^identifier.type.text = "Accession ID"
+* ^identifier.type = $0203#ACSN "Accession ID"
+* $testResultsValueURl#9860243
+* $testResultsValueURl#9860241
+
+
+
+ValueSet: EntryPointForCommunityLevelTesting
+Id: HIV.D.DE5
+Title: "Entry point for community-level testing"
+Description: "0.1.1"
+* ^name = "Entry point for community-level testing"
+* ^version = "0.1.1"
+* ^status = #active
+* ^meta.lastUpdated = "2024-07-02T09:13:39.228191Z"
+* ^identifier.system = "https://api.staging.openconceptlab.org"
+* ^identifier.value = "/orgs/Kenya_SMART_Guidelines/ValueSet/HIV.D.DE5/"
+* ^identifier.type.text = "Accession ID"
+* ^identifier.type = $0203#ACSN "Accession ID"
+* $kenyaValueSetURlValue#9860237
+* $kenyaValueSetURlValue#9860239
+* $kenyaValueSetURlValue#community-based
+* $kenyaValueSetURlValue#other-community-based-testing
+
+
+ValueSet: TransferIn
+Id: transfer.in
+Title: "Transfer in"
+Description: "0.1.1"
+* ^name = "Transfer in"
+* ^version = "0.1.1"
+// * ^url = "https://app.staging.openconceptlab.org/#/orgs/Kenya_SMART_Guidelines/collections/transfer_in"
+* ^status = #active
+* ^meta.lastUpdated = "2024-07-02T09:14:51.685596Z"
+* ^identifier.system = "https://api.staging.openconceptlab.org"
+* ^identifier.value = "/orgs/Kenya_SMART_Guidelines/ValueSet/transfer.in/"
+* ^identifier.type.text = "Accession ID"
+* ^identifier.type = $0203#ACSN "Accession ID"
+* $transferInValueSetURlValue#1065
+* $transferInValueSetURlValue#1066
+* $transferInValueSetURlValue#9860127
+* $transferInValueSetURlValue#9860129
+
+Alias: $0203 = http://hl7.org/fhir/v2/0203
+
+ValueSet: HTSSetting
+Id: HTS.setting
+Title: "HIV Testing Services Setting"
+Description: "0.1.1"
+* ^name = "HTS Setting"
+* ^version = "0.1.1"
+// * ^url = "https://app.staging.openconceptlab.org/#/orgs/Kenya_SMART_Guidelines/collections/HTS_setting/"
+* ^status = #active
+* ^meta.lastUpdated = "2024-07-02T09:15:25.951956Z"
+* ^identifier.system = "https://api.staging.openconceptlab.org"
+* ^identifier.value = "/orgs/Kenya_SMART_Guidelines/ValueSet/HTS_setting/"
+* ^identifier.type.text = "Accession ID"
+* ^identifier.type = $0203#ACSN "Accession ID"
+* $htsSettingValueSetUrlValue#163488
+* $htsSettingValueSetUrlValue#1537
+
+
+ValueSet: EntryPoint
+Id: entry.point
+Title: "Entry Point"
+Description: "0.1.0"
+* ^name = "Entry Point"
+* ^version = "0.1.0"
+// * ^url = "https://app.staging.openconceptlab.org/#/orgs/Kenya_SMART_Guidelines/collections/entry_point/"
+* ^status = #active
+* ^meta.lastUpdated = "2024-05-28T19:27:57.377923Z"
+* ^identifier.system = "https://api.staging.openconceptlab.org"
+* ^identifier.value = "/orgs/Kenya_SMART_Guidelines/ValueSet/entry_point/"
+* ^identifier.type.text = "Accession ID"
+* ^identifier.type = $0203#ACSN "Accession ID"
+* include codes from system $entryPointValueSetUrlValue|0.1.0
+    where excludeWildcard = "true" and
```

Comment:
Note that we decided on our call today that these filter-focused "where" lines come from how OCL builds its intensional value set references. An alternative option is to add the Entry Point codes manually to the appropriate OCL collection, which will turn this value set into an extensional one instead of an intensional one. Happy to walk through this process with you if needed!